sentinel-core/src/main/java/com/alibaba/csp/sentinel/util/ConfigUtil.java

```diff
@@ -108,18 +114,11 @@ private static Properties loadPropertiesFromClasspathFile(String fileName) {
         Properties properties = new Properties();
         for (URL url : list) {
             try {
```

Comment:
Can we remove the redundant `try`? Just one try-catch is enough.

Reply:
👌